target/linux/ipq40xx/config-5.4

```diff
@@ -414,6 +348,7 @@ CONFIG_PINCTRL_MSM=y
 # CONFIG_PINCTRL_SDM660 is not set
 # CONFIG_PINCTRL_SDM845 is not set
 # CONFIG_PINCTRL_SM8150 is not set
+CONFIG_PLUGIN_HOSTCC="g++"
```

Comment:
Please manually drop the CONFIG_PLUGIN_HOSTCC again ...

Reply:
Done